openfisca_us/variables/finance/benefit/tanf.py

```diff
@@ -38,6 +38,12 @@ class tanf_max_amount(Variable):
     documentation = "The maximum benefit amount a family could receive from Temporary Assistance for Needy Families given their state and family size."
     unit = "currency-USD"
 
+    def formula(spm_unit, period, parameters):
+        family_size = spm_unit("family_size", period)
```

Comment:
Nice, could you add an `spm_unit_num_people` OpenFisca variable if that's what this is?

Reply:
Should we do that or just use `spm_unit.nb_persons()`?

Reply:
Good point, that'd work

Reply:
Resolving but also pointing to #324